app/controllers/dashboards_controller.rb

```diff
@@ -1,5 +1,73 @@
 # frozen_string_literal: true
 
 class DashboardsController < ApplicationController
-  def show; end
+  def show
+    @latest_issues = Issue.not_archived.order(created_at: :desc).limit(10)
+    @own_issues = own_issues
+    @former_issues = former_issues
+    notices
+    @notices_count = @received_not_accepted.count + @open_ideas.count + @not_approved_issues.count +
+                     @in_process.count + @open_not_accepted.count
+  end
+
+  private
+
+  def notices
+    @received_not_accepted = received_not_accepted
+    @open_ideas = open_ideas(Time.current - 60.days)
+    @not_approved_issues = Issue.not_archived.not_approved
+    @in_process = in_process(Time.current - 30.days)
+    @open_not_accepted = open_not_accepted(Time.current - 3.days)
+  end
+
+  def own_issues
+    Issue.not_archived.joins(:all_log_entries, category: :parents).where(
+      category: { deleted: false }, parents_category: { deleted: false },
+      status: %w[pending received reviewed in_process not_solvable closed],
+      log_entry: { attr: [nil] + %w[address status description kind] }
+    ).where(LogEntry.arel_table[:created_at].gteq(Date.current - 7.days)).limit(10).distinct
+  end
+
+  def former_issues
+    Issue.joins(:all_log_entries, category: :parents).not_archived.where(
+      category: { deleted: false }, parents_category: { deleted: false },
+      archived_at: nil
+    ).where.not(responsibility_id: Current.user.group_ids).where(changed_responsibilities).limit(10).distinct
+  end
+
+  def changed_responsibilities
+    <<~SQL.squish
+      (
+        SELECT COUNT(le.created_at) FROM log_entry le
```

Comment:
Tabellen-Namen bitte nicht fest einschreiben. Bitte in LogEntry.table_name ändern.